docs/new_services.rst

```diff
@@ -0,0 +1,21 @@
+.. _topics-new_services:
+
+New Services
```

Comment:
Maybe call it "Writing a new service" or "Writing new services"? Somehow, "New Services" title was confusing to me.